logs/spring.log

```diff
@@ -0,0 +1,1669 @@
+2022-07-16 16:03:28.630  INFO 17740 --- [SpringApplicationShutdownHook] j.LocalContainerEntityManagerFactoryBean : Closing JPA EntityManagerFactory for persistence unit 'default'
```

Comment:
Please make sure to not push files like these to git, logs files even should be out of the project.

Please think about that and make the needed changes.